frontend/public/scss/dashboard.scss

```diff
@@ -29,7 +28,8 @@ $program-badge-bg: #DBF2F3;
     }
 
     h2 a {
-      text-decoration: underline;
+      font-size: 20px;
```

Comment:
Nit: this `font-size` I think should apply to `h2` without an `a` - otherwise the default font size of 28px is used when there's no link in the `h2`. (This only happens if there's no courseware URL for a course, which is admittedly an edge case.)